content/_partners/faq._en.md

```diff
@@ -19,52 +19,48 @@ general_accordion:
 
             - Troubleshooting after launch and user support
```

Comment:
The [content page](https://docs.google.com/document/d/1NTqpFidS9XBCJ5xFiZ-G2YGb8SilACVLaOeJn7eCkrc/edit#) indicates that we should add "Technical" to the front of this phrase